backend/arm64/proc.ml

```diff
@@ -91,7 +91,7 @@ let phys_reg ty n =
        allocator relies on. It is safe to guard it with this flag since the LLVM
        backend doesn't get that far. *)
     if !Clflags.llvm_backend
-    then { hard_int_reg.(n) with typ = ty }
+    then Reg.create_alias hard_int_reg.(n) ~typ:ty
```

Comment:
(That does not really matter, but
we could have the same sequence
in both `proc.ml` files.)

Reply:
I am not exactly sure why I wrote "sequence",
but I meant we should probably apply the
same diff to https://github.com/oxcaml/oxcaml/blob/main/backend/arm64/proc.ml#L94.

Reply:
Ah, I see, I added  `let r = hard_int_reg.(n) in`  in amd64/proc.ml but not in `arm64/proc.ml`. We will remove the condition soon.